Tasks/FtpUpload/task.json

```diff
@@ -76,6 +77,7 @@
         },
         {
             "name": "rootFolder",
+            "aliases": [ "rootDirectory" ],
```

Comment:
I wonder if this should be "sourceDirectory" since the label says "source"

Reply:
root is probably more accurate.  I wonder if we should change the label.